src/test/ui/issues/issue-67037-pat-tup-scrut-ty-diff-less-fields.stderr

```diff
@@ -17,6 +17,15 @@ LL | struct P<T>(T); // 1 type parameter wanted
 ...
 LL |     let P() = U {};
    |         ^^^ expected 1 field, found 0
+   |
+help: use `_` to explicitly ignore each field
+   |
+LL |     let P(_) = U {};
+   |           ^
+help: use `..` to ignore all unmentioned fields
+   |
+LL |     let P(..) = U {};
+   |           ^^
```

Comment:
Or maybe we shouldn't even show this up when there is one field so rustfix can apply the suggestion?

Reply:
I agree, we shouldn't suggest `..` when there's only a single field left (including when the current code is `P(a)` and the suggestion is for `P(a, _)`, but not if the current code is `P(_)`, then the suggestion should be for `P(..)`).

Reply:
I would rather not make this change too complex. Do you think we could do that in a followup PR?

Reply:
No, I believe it is simple enough to do (a simple one) in this PR. Let me send a suggestion for that.